Week4/README.md

```diff
@@ -0,0 +1,26 @@
+# Reading Material Databases Week 4
+
+## Agenda
+
+These are the topics for week 4, all in MongoDB:
+
+- [Embedded vs Normalised data modeling](https://hackyourfuture.github.io/study/#/databases/nosql/mongodb/embedded-vs-normalised.md)
```

Comment:
Do we need a `LESSONPLAN.md` for week 4 ?
If I look at the study material forNoSQL and Mongo, it is hard to follow with the example of `collection_name` 

We follow the example of `employee-departments-projects` tables through out the lesson plans and I would continue creating MongoDB `documents/collections` for one chosen database (be it a movie database or projects database)

Reply:
Will do this in a new PR before week 4 starts. Good point, I forgot about it